app/models/user.rb

```diff
@@ -1,6 +1,6 @@
 class User < ApplicationRecord
-  # Include default devise modules. Others available are:
-  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
+
+  validates_presence_of :firstname, :lastname, :address, :drivingage, :age
```

Comment:
I want you to change the attributes `firstname, lastname and drivingage` to `driving_age` etc

Reply:
done